Solutions/MongoDBAudit/DataConnectors/Connector_MongoDBAudit.json

```diff
@@ -0,0 +1,166 @@
+                    "parameters": {
+                    "title": "Choose where to install the Windows agent:",
+        "instructionSteps": [
+                {
+                "title": "Install agent on Azure Windows Virtual Machine",
+                "description": "Select the machine to install the agent on and then click **Connect**.",
+                "instructions": [
+                    {
+                    "parameters": {
+                    "linkType": "InstallAgentOnVirtualMachine"
+                    },
+                    "type": "InstallAgent"
+                    }
+                ]
+            },
+                {
+                "title": "Install agent on a non-Azure Windows Machine",
+                "description": "Download the agent on the relevant machine and follow the instructions.",
+                "instructions": [
+                    {
+                    "parameters": {
+                    "linkType": "InstallAgentOnNonAzure"
+                    },
+                    "type": "InstallAgent"
+                    }
+                ]
+            }
+        ]
+            },
+                "type": "InstructionStepsGroup"
+                }
+            ]
+                },
+    {
+                    "title": "2. Configure MongoDBAudit to write logs to files",
+                    "description":"Edit mongod.conf file (for Linux) or mongod.cfg (for Windows) to write logs to files:\n\n>**dbPath**: data/db\n\n>**path**: data/db/auditLog.json\n\nSet the following parameters: **dbPath** and **path**. Refer to the [MongoDB documentation for more details](https://www.mongodb.com/docs/manual/tutorial/configure-auditing/)"
+                },
+    {
+            "title": "3. Configure the logs to be collected",
+            "description": "Configure the custom log directory to be collected" ,
+            "instructions": [
+                {
+                    "parameters": {
+                        "linkType": "OpenCustomLogsSettings"
+                    },
+                    "type": "InstallAgent"
+                }
+            ]
+        },
+    {
+        "title": "",
+        "description":"1. Select the link above to open your workspace advanced settings \n2. From the left pane, select **Settings**, select **Custom Logs** and click **+Add custom log**\n3. Click **Browse** to upload a sample of a MongoDBAudit log file. Then, click **Next >**\n4. Select **Timestamp** as the record delimiter and click **Next >**\n5. Select **Windows** or **Linux** and enter the path to MongoDBAudit logs based on your configuration \n6. After entering the path, click the '+' symbol to apply, then click **Next >** \n7. Add **MongoDBAudit** as the custom log Name (the '_CL' suffix will be added automatically) and click **Done**."
+    },
+    {
+        "title": "Validate connectivity",
+        "description":"It may take upwards of 20 minutes until your logs start to appear in Microsoft Sentinel."
+    }
+]
+}
```

Comment:
Also include Metadata section as provided in "https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_Syslog_template.json".

Reply:
Done